examples/gno.land/p/nt/commondao/proposal.gno

```diff
@@ -97,6 +97,12 @@ type (
 		Execute() error
 	}
 
+	// RenderableProposal makes sure that the proposal is renderable
+	RenderableProposal interface {
+		ProposalDefinition
+		Render() string
+	}
```

Comment:
Could be simpler:

```suggestion
	// Renderable makes sure that the proposal is renderable.
	Renderable interface {
		Render() string
	}
```

This interface might be a good candidate to be defined in a `p/nt/commondao/ui` package.